onnx/defs/tensor/defs.cc

```diff
@@ -3890,4 +3890,78 @@ ONNX_OPERATOR_SET_SCHEMA(
 	        }
         }));
 
+static const char* Col2Im_ver16_doc = R"DOC(
+The operator rearranges the input matrix columns into blocks.
+
+Col2Im behaves like PyTorch's fold for two output spatial dimensions https://pytorch.org/docs/stable/generated/torch.nn.Fold.html
```

Comment:
will this op support input >4D?

Reply:
My understanding is that it doesnt

Reply:
pytorch clearly states it does not... so if we want to keep the behavior same then let's clarify it and add a check around this in type and shape inference function

Reply:
Added "Note that only 4-D output tensors (batched image-like tensors) are supported."